docs/customservices.md

```diff
@@ -45,7 +45,7 @@ The following configuration is available for the OpenWeatherMap service:
 - name: "Weather"
   location: "Amsterdam" # your location.
   locationId: "2759794" # Optional: Specify OpenWeatherMap city ID for better accuracy
-  apikey: "<---insert-api-key-here--->" # insert your own API key here. Request one from https://openweathermap.org/api.
+  apikey: "<MY-SUPER-SECRET-API-KEY>" # insert your own API key here. Request one from https://openweathermap.org/api.
```

Comment:
While I appreciate the effort to make more consistent documentation, let's maintain proper change cleanliness by reverting the unrelated changes (as far as I can tell, all changes in this file).